### ISSUE TITLE
Did u use validation set for hyperparameters tunning

### ISSUE THREAD

**Issue by hzhz2020** [google-research/fixmatch#83]
Hi 

I would like to ask a clarification question regarding experiment setting, i'm curious did u tune the hyperparameter for FixMatch or the other compared baselines?
